tests/5.0/requires/test_requires_dynamic_allocators.c

```diff
@@ -0,0 +1,64 @@
+//===------ test_requires_dynamic_allocators.c ----------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// Tests the requires dynamic allocators clause by working with an OMP
+// allocator inside a target region. Allocators testing is based on the
+// OpenMP 5.0 example for allocators. The allocator testing first creates
+// an allocator, with 64-byte alignment and the default memory space,
+// then checks that 64-byte alignment is correct and that the memory can
+// be written to in the target region. The tests checks that the values
+// were written correctly, and then frees the memory and deletes the
+// allocator.
+//
+//===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1024
+
+int test_requires() {
+  int errors = 0;
+
+#pragma omp target map(from: errors)
+  {
+    int* x;
+    omp_memspace_t x_memspace = omp_default_mem_space;
+    omp_alloctrait_t x_traits[1] = {omp_atk_alignment, 64};
```

Comment:
Remove this for now. Don't test alignment.

Reply:
I looked into this a bit further to see exactly where this came from. Allocator traits like alignment are explained on page 153, but the name `omp_atk_alignment` comes from page 406. It is a member of the allocator keys enum. `omp_alloctrait_t` is the type `omp_init_allocator` expects as its third argument.